swarm/src/handler.rs

```diff
@@ -94,9 +94,9 @@ pub use select::{ConnectionHandlerSelect, IntoConnectionHandlerSelect};
 /// continue reading data until the remote closes its side of the connection.
 pub trait ConnectionHandler: Send + 'static {
     /// Custom event that can be received from the outside.
-    type InEvent: fmt::Debug + Send + 'static;
+    type FromBehaviour: fmt::Debug + Send + 'static;
     /// Custom event that can be produced by the handler and that will be returned to the outside.
-    type OutEvent: fmt::Debug + Send + 'static;
+    type ToBehaviour: fmt::Debug + Send + 'static;
```

Comment:
Can we update the docs here please?

Reply:
@thomaseizinger Done, I updated the doc comment above the type events declaration to be in agreement with the new namings.